server/acl_casbin_policy_dba.csv

```diff
@@ -85,3 +85,5 @@ p, DBA, /savedquery, GET
 p, DBA, /savedquery/{id}, GET
 p, DBA, /savedquery/{id}, PATCH_SELF
 p, DBA, /savedquery/{id}, DELETE_SELF
+p, DBA, /license, GET
```

Comment:
I'm not sure which role has access for `/license`, but I found all roles can access /plan APIs, and the models for /plan defined here are not used, cause we just ignore the check in ACL middleware and JWT middleware.

Reply:
All roles should be able to read, but updating license should only be done by admin as you did.